Makefile

```
@@ -11,28 +15,24 @@ all: build
 # -------------------
 
 ifneq ($(NIX),1)
-POETRY     := poetry -C pykwasm
-POETRY_RUN := $(POETRY) run --
-KDIST      := $(POETRY_RUN) kdist
+UV     := uv --directory pykwasm
```

Consider

```suggestion
UV     := uv --project pykwasm
```

Then its possible the `tests/*` targets will work without the `../`.